tests/check/check_signed_video_auth.c

```diff
@@ -653,8 +653,8 @@ START_TEST(modify_one_sei)
   //           IPPSI   ->   (valid) -> ....P
   // One pending NAL Unit per GOP. Note that a modified 'I' affects two GOPs due to linked hashes,
   // but it will also affect a third if we validate with a gop_hash.
-  struct validation_stats expected = {.valid_gops = 2,
-      .invalid_gops = 2,
+  struct validation_stats expected = {.valid_gops = 3,
```

Comment:
Can you elaborate on why you get more valid GOPs and update the description above, since that indicate there are only 2 valid GOPs.